### ISSUE TITLE
vertical scrollbar, virtual scrolling & rowdetail auto height: disappearing rows

### ISSUE THREAD

**Issue by rauwebieten** [swimlane/ngx-datatable#1657]
**I'm submitting a ...** 
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
Enable vertical scrollbar and virtual scrolling, and rowdetail with rowheight set to auto.

When expanding rowdetail, all following rows disappear

**Expected behavior**
rows should stay visible, rowdetail should push them lower.

**Reproduction of the problem**
https://stackblitz.com/edit/angular-ngx-datatable-row-grouping-and-row-detail-mjqjrd

**What is the motivation / use case for changing the behavior?**
I need a fixed-height table, with virtual scrolling and a rowdetail with variable height (depending on the row data)

**Please tell us about your environment:**
see https://stackblitz.com/edit/angular-ngx-datatable-row-grouping-and-row-detail-mjqjrd

* **Table version:** 
14.0.0

* **Angular version:** 
7.2.6

* **Browser:** 
all

* **Language:** 
all

**Comment by samir-kamble** [comment]
Bump!! Facing Same issue. Can see the same in DEMO of this library as well.

**Comment by sacgrover** [comment]
@swimlane
 I have fixed this issue for following issue. PR has been raised:
Row-details - bug on the official example #1793

**Comment by marjan-georgiev** [comment]
Resolved by #1799
